### ISSUE TITLE
When mocha runs in parallel mode, the failed test cases appear yellow instead of red.

### ISSUE THREAD

**Issue by jamesmortensen** [allure-framework/allure2#1739]
**Describe the bug**

When we run tests, any test which fails due to an AssertionError is reported in Allure in _red_, as a failing test.  Any other thrown Error is reported as a broken test, in yellow.

While this functionality works as expected in sequential mode, failed tests are reported as broken tests (in yellow) when run in parallel.

**To Reproduce**
1. Clone this repo: https://github.com/jamesmortensen/allure-js-reporter-issues#when-run-in-parallel-mode-with-hooks-hookerror-is-not-a-function and follow the instructions to run the tests and generate the report.

The example has us run tests in sequential mode as well as parallel mode.  The README also has screenshots which show the issue.

**Expected behavior**

**Screenshots**

- Good (shows red):  https://github.com/jamesmortensen/allure-js-reporter-issues/blob/master/screenshots/sequential-mode-report.png
- Bad (shows yellow):  https://github.com/jamesmortensen/allure-js-reporter-issues/blob/master/screenshots/parallel-mode-report.png

**Environment (please complete the following information):**

| Allure version | 2.9.0 | (not sure what this is - see below for details)
| --- | --- |
| Test framework | mocha@10.0.0 | chai@4.3.6
| Allure adaptor | allure-mocha@2.0.0-beta.19 | mocha-multi-reporters@1.5.1
| Generate report using | allure-commandline@2.18.1 |

**Additional context**

**Comment by jamesmortensen** [comment]
Duplicate of https://github.com/allure-framework/allure-js/issues/485
